submissions/opeyemiPeter/Javascript/task-2.js

```diff
@@ -0,0 +1,19 @@
+const diff = 123 - 7;
+var  num1 = 1;
+
+while (num1 < diff) {
+    switch (true) {
+      case ((num1 % 3) === 0):
+            console.log("Software");
+            break;
+        case ((num1 % 5) === 0):
+            console.log("Developer");
+            break;
+        case ((num1 % 3 === 0) && (count % 5 === 0)):
```

Comment:
This case should come first @opeyemipeter1759 As it is right now you won't be able to log `software developer` because the first 2 cases will be true and will pass by the third case.
Also, there is no variable `count`